### DISCUSSION TITLE
Project save folder - stop iterating (i.e. predict,predict2,predict3..)

### DISCUSSION THREAD

**Discussion by Natriumpikant** [ultralytics/ultralytics#1118] [Q&A] [answered]
Good morning,

i want to iterate through an image-folder (not with the integraded YOLO function).

If i process multiple images with the following code:

`results = model.predict(imgsz=1024, source=image_path, save=True, project= "/home/hans/save/") `

it will save every image to a separate folder, for example:
- /home/hans/save/predict
- /home/hans/save/predict2
- /home/hans/save/predict3
- ...

How to achieve that every image will be saved in the same folder (/home/hans/save/predict)

Thanks for an advice
Regards

**Comment by Laughing-q** [comment, marked as answer]
@Natriumpikant add `exist_ok=True`

**Reply by Natriumpikant** [reply]
Thanks!

**Reply by glenn-jocher** [reply]
@Natriumpikant you're welcome! If you have any more questions or need further assistance, feel free to ask. Happy coding! 😊🚀
